dbms/src/Flash/Pipeline/Schedule/Tasks/EventTask.cpp

```diff
@@ -60,40 +47,23 @@ void EventTask::finalizeImpl() noexcept
     {
         exec_status.onErrorOccurred(std::current_exception());
     }
+    assert(event);
```

Comment:
```suggestion
    RUNTIME_CHECK(event);
```

Reply:
Since all event_ptr's must be non-nullptr, I've used assert here, maybe delete the useless assert's?

Reply:
removed.